src/main/resources/org/sagebionetworks/web/client/widget/accessrequirements/TermsOfUseAccessRequirementWidgetViewImpl.ui.xml

```diff
@@ -7,6 +7,11 @@
 		type='org.sagebionetworks.web.client.IconsImageBundle' />
 	<b:Panel marginBottom="0">
 		<b:PanelBody>
+			<bh:Div ui:field="accessRequirementIDUI" visible="false">
+				<g:InlineLabel addStyleNames="boldText margin-right-5"
+							   text="Click Wrap ID:" />
```

Comment:
Click-wrap ARs share the same ID space as other AR types, so let's label the ID so as not to mislead ACT members into thinking that they are separate.
```suggestion
							   text="Access Requirement ID:" />
```

Reply:
I asked Ann about this and she wanted it labeled as Click Wrap but if you think Access Requirement ID is better I can go with it.

Reply:
If that's just to make it more clear to ACT that this it's a click wrap, maybe add a line above the ID that says "Click Wrap". It's not a big deal, so I'm ok with Click Wrap ID or whatever makes the most sense